src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java

```diff
@@ -67,14 +70,13 @@ void shouldReturnExpectedSubmittedCallbackResponseObject() {
         SubmittedCallbackResponse response = (SubmittedCallbackResponse) handler.handle(params);
 
         String documentLink = "https://www.google.com";
-        String responsePackLink = "https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf";
         LocalDateTime serviceDeadline = LocalDate.now().plusDays(112).atTime(23, 59);
         String formattedServiceDeadline = formatLocalDateTime(serviceDeadline, DATE_TIME_AT);
 
         String body = format(
             "<br />Follow these steps to serve a claim:"
                 + "\n* [Download the sealed claim form](%s) (PDF, 123KB)"
-                + "\n* Send the form, particulars of claim and [a response pack](%s) (PDF, 266 KB) "
+                + "\n* Send the form, particulars of claim and [a response pack](%s target=\"_blank\") (PDF, 266 KB) "
```

Comment:
Does this actually work?

Reply:
yes it does

Reply:
It doesn't work in incognito mode of chrome but works in normal mode of chrome.

Reply:
Replacing this with href ta as this doesn't work with Safari either.